examples/tfengine/devops.hcl

```diff
@@ -27,23 +28,24 @@ template "devops" {
     # Run `terraform init` in the bootstrap module to backup its state to GCS.
     # enable_bootstrap_gcs_backend = true
 
-    project_id       = "example-devops"
-    state_bucket     = "example-terraform-state"
-    storage_location = "us-central1"
-    admins_group     = "example-org-admin@example.com"
-    project_owners = [
-      "group:example-devops-owners@example.com",
-    ]
-
-    enable_terragrunt = false,
+    admins_group      = "example-org-admin@example.com"
+    state_bucket      = "example-terraform-state"
+    enable_terragrunt = false
 
+    project = {
+      project_id = "example-devops"
```

Comment:
How come the "project" prefix stays for id, but not for owners?

Reply:
I stuck with using project_id (and not 'id') as that is what the underlying terraform resource uses: https://www.terraform.io/docs/providers/google/r/google_project.html.

Same with BQ dataset. The resource calls the main id 'dataset_id'. So I prefer to be consistent with the API.